analyzer/src/main/kotlin/managers/utils/NuGetSupport.kt

```diff
@@ -304,7 +311,8 @@ fun PackageManager.resolveNuGetDependencies(
     val issues = mutableListOf<OrtIssue>()
 
     val references = reader.getPackageReferences(definitionFile)
-    support.buildDependencyTree(references, dependencies, packages, issues)
+    val ids = references.map { Identifier(type = "NuGet", namespace = "", name = it.name, version = it.version) }
```

Comment:
Maybe implement the creation of an `Identifier` from a `NutGetDependency` as an extension function on the latter?